packages/firebase_dynamic_links/README.md

```diff
@@ -117,24 +118,38 @@ class MyHomeWidgetState extends State<MyHomeWidget> {
   @override
   void initState() {
     super.initState();
-    _retrieveDynamicLink();
+    this.initDynamicLinks();
   }
 
-  Future<void> _retrieveDynamicLink() async {
-    final PendingDynamicLinkData data = await FirebaseDynamicLinks.instance.retrieveDynamicLink();
+  void initDynamicLinks() async {
```

Comment:
Seems like this should be `Future<void>` right?

If it doesn't need to be a public method, using an underscore seems more appropriate

Reply:
Fixed in the latest commit: https://github.com/flutter/plugins/pull/1687/commits/b5c45c262941f09ed00b90c4076b7fbfd67871dc